script.js

```diff
@@ -10,34 +13,45 @@ function computerSelection() {
     return randomComputerSelection;
 }
 
-function playRound() {
-    //let playerInput =  prompt("Choose Rock, Paper, or Scissors!");
-    //let playerSelection = playerInput.toLowerCase();
-    //let computersMove = computerSelection();
-
-    if (
+//plays a single round and produces round winner
+function playRound(playerSelection) {
+    let computersMove = computerSelection();
+        if (
         (playerSelection === choiceRock && computersMove === choiceScissors) ||
         (playerSelection === choiceScissors && computersMove === choicePaper) ||
         (playerSelection === choicePaper && computersMove === choiceRock)) {
-        return `You won! ${playerSelection.toUpperCase()} beats ${computersMove.toUpperCase()}.`;
+        document.querySelector("h2").innerHTML = `Player 1 wins! ${playerSelection.toUpperCase()} beats ${computersMove.toUpperCase()} `;
```

Comment:
If someone added another H2 to your page that would break your code. 

Use an id as a selector in this case.